hyrax.gemspec

```diff
@@ -37,8 +37,7 @@ SUMMARY
   spec.add_dependency 'blacklight', '~> 6.14'
   spec.add_dependency 'blacklight-gallery', '~> 0.7'
   spec.add_dependency 'breadcrumbs_on_rails', '~> 3.0'
-  # Pin browse-everything to stable version
-  spec.add_dependency 'browse-everything', '< 0.16'
+  spec.add_dependency 'browse-everything', '~> 1.0'
```

Comment:
I'm not sure it's okay for us to force this upgrade as part of the 2.x release series. Thoughts?

Reply:
Certainly, I can adjust this PR.